docs/quick_start.md

```diff
@@ -0,0 +1,88 @@
+# Quick start
+
+## Use sc-machine as a C++ library in your project
+
+### Conan
+
+You can use Conan to install sc-machine. To do this you need to create `conanfile.txt` in the root of the project and add to it:
+
+```ini
+[requires]
+sc-machine/<version>
+```
+
+Then run the following command in the project root:
+
+```sh
+conan install .
+```
+
+Import sc-machine targets into your CMake project by using:
+
+```cmake
+find_package(sc-machine REQUIRED)
+```
+
+Start building! Refer to our [C++ Guide](https://ostis-ai.github.io/sc-machine/sc-memory/api/cpp/guides/simple_guide_for_implementing_agent/) on how to quickly develop an sc-machine agent in C++ from scratch.
+
+### GitHub Releases
+You can download pre-built artifacts from [GitHub Releases](https://github.com/ostis-ai/sc-machine/releases). Extract it to any location, then make it available to CMake by appending folder path to `CMAKE_PREFIX_PATH`:
+
+```cmake
+set(FABULOUS_PROJECT_SC_MACHINE_PATH "/path/to/extracted/sc-machine" 
+    CACHE PATH "sc-machine installation path"
+)
+
+list(APPEND CMAKE_PREFIX_PATH ${FABULOUS_PROJECT_SC_MACHINE_PATH})
+```
+
+Find more info on installation methods on the [build system page](https://ostis-ai.github.io/sc-machine/build/build_system/) (including some more advanced snippets for CMake).
+
+## Install sc-machine as a service
+
+It's possible to use sc-machine through a WebSocket API. Docs for our WebSocket API clients in [Python](https://github.com/ostis-ai/py-sc-client) and [TypeScript](https://github.com/ostis-ai/ts-sc-client) are available in the respective repositories. You might want to check [sc-web](https://github.com/ostis-ai/sc-web) for a more user-friendly interface as well.
+
+### Docker
+
+We provide a Docker image for this project. Here's a quick snippet on how to launch it.
+
+```sh
+# clone the repo
+git clone https://github.com/ostis-ai/sc-machine
+cd sc-machine
+# create empty knowledge base sources folder
+mkdir kb && cat "." > kb/repo.path
+# note: at this stage you can move your KB sources to the ./kb folder
+
+# build knowledge base
+docker compose run --rm machine build
```

Comment:
Maybe, you have built other sc-machine container

Reply:
Try to remove them and repeat this commands

Reply:
something goes wrong, looks like sc-machine from dockerhub is used, it has sc-builder call on line 28
![image](https://github.com/user-attachments/assets/fa1350d5-601e-4692-8201-276c941b65ba)

Reply:
It'll work fine, but after the release. It doesn't work now, because there is no 0.10.0 machine image on Docker Hub. To check this, you can run `docker compose build` before `docker compose run --rm machine build`

Reply:
The sc-machine repo provides docker-compose.yml, where all the necessary arguments are customised for image `sc-machine`. In addition, the image version is also listed there, you can always change it.

To avoid cloning the repository, you'll have to do the following commands:

`docker pull ostis/sc-machine:latest
docker run --rm -it -v <full path to kb sources>:/kb -v sc-machine_kb:/kb.bin -e <env settings> ostis/sc-machine:latest build /kb
 docker run --rm -it -v sc-machine_kb:/kb.bin -p 8090:8090 ostis/sc-machine:latest serve
`

I believe that the way the sc-machine is started in the docker that is currently specified is more pleasant to use and more appropriate for QuickStart

Reply:
looks like devcontainer documentation is not the same as documentation to how build and run local sc-machine files
can you add somewhere that info? if I want to change local sc-machine files and run in docker and see what changes